src/components/ha-cover-controls.html

```diff
@@ -17,12 +17,12 @@
     <div class='state'>
       <paper-icon-button icon="mdi:arrow-up" on-tap='onOpenTap'
         invisible$='[[!entityObj.supportsOpen]]'
-        disabled='[[entityObj.isFullyOpen]]'></paper-icon-button>
+        disabled='[[entityObj.disableButtonFullyOpen]]'></paper-icon-button>
       <paper-icon-button icon="mdi:stop" on-tap='onStopTap'
         invisible$='[[!entityObj.supportsStop]]'></paper-icon-button>
       <paper-icon-button icon="mdi:arrow-down" on-tap='onCloseTap'
         invisible$='[[!entityObj.supportsClose]]'
-        disabled='[[entityObj.isFullyClosed]]'></paper-icon-button>
+        disabled='[[entityObj.disableButtonFullyClosed]]'></paper-icon-button>
```

Comment:
Don't add this to entityObj. Compute it in component instead: `disabled='[[computeCloseDisabled(stateObj, entityObj]]'